worlds/alttp/SubClasses.py

```diff
@@ -18,20 +18,18 @@ def __init__(self, player: int, name: str = '', address=None, crystal: bool = Fa
 
 class ALttPItem(Item):
     game: str = "A Link to the Past"
+    type: Optional[str]
+    _pedestal_hint_text: Optional[str]
+    _hint_text: Optional[str] = None
     dungeon = None
 
-    def __init__(self, name, player, classification=ItemClassification.filler, type=None, item_code=None, pedestal_hint=None,
-                 pedestal_credit=None, sick_kid_credit=None, zora_credit=None, witch_credit=None,
-                 flute_boy_credit=None, hint_text=None):
+    def __init__(self, name, player, classification=ItemClassification.filler, type=None, item_code=None,
+                 pedestal_hint=None, **kwargs):
```

Comment:
personally not a fan of **kwargs, as it makes it harder to see args like "hint_text" being available at a glance

Reply:
would you prefer the unused variable names being there? It is easier to read this way, and the whole NamedTuple is passed as a dict, so we have to "catch" the other ones, but i don't mind reverting it.

Reply:
Thinking about it, we can also move hint_text out of kwargs**. This changes the order of arguments, but since this is always called with a dict, order does not matter.